pkg/model/iam/iam_builder.go

```diff
@@ -49,26 +50,26 @@ const PolicyDefaultVersion = "2012-10-17"
 // Policy Struct is a collection of fields that form a valid AWS policy document
 type Policy struct {
 	clusterName               string
-	unconditionalAction       sets.String
-	clusterTaggedAction       sets.String
-	clusterTaggedCreateAction sets.String
+	UnconditionalAction       sets.String `json:"-"`
+	ClusterTaggedAction       sets.String `json:"-"`
+	ClusterTaggedCreateAction sets.String `json:"-"`
```

Comment:
Perhaps only expose receivers that insert into these? I don't think we want external modules to be able to remove permissions from a policy.

Reply:
That makes sense. Will do that.